spec/shortcuts_spec.rb

```diff
@@ -159,6 +159,39 @@
     end
   end
   
+  context "::magic" do
+  
+    [:int64,:int32,:int16,:int8].each do |dtype|
+      context dtype do  
+        it "creates a matrix with numbers from 1 to n^n(n squared)" do
+          a = NMatrix.magic(3)
```

Comment:
I have made the required changes. 
@v0dro Is there any need to check for all the dtypes as we are working with only integers over here?